corehq/form_processor/tests/test_form_dbaccessor.py

```diff
@@ -91,6 +91,10 @@ def test_get_forms_by_last_modified(self):
 
     def test_get_with_attachments(self):
         form = create_form_for_test(DOMAIN)
+        form = FormAccessorSQL.get_form(form.form_id)  # refetch to clear cached attachments
+        with self.assertNumQueries(1, using=db_for_read_write(XFormAttachmentSQL)):
```

Comment:
Why does this happen twice? I was originally thinking we should just change it to 0 queries expected

Reply:
in the original version it should have been changed to 0 since it would use the 'cached' attachments but now that I've re-fetched the form from the DB it needs to do another query to get the attachments.

The reason I did that instead of changing it to `0` is that it the test is making sure that it doesn't query the DB if you call `get_form_with_attachments`. So asserting that it will query the DB if you just call `get_form` is part of the test.